cli_li3ds/import_platform.py

```diff
@@ -85,3 +97,29 @@ def take_action(self, parsed_args):
         objs.add(pconfig)
         objs.get_or_create()
         self.log.info('Success!\n')
+
+    @staticmethod
+    def read_lidar_rigid_transform_params(filename):
+        config = configparser.ConfigParser()
+        config.read(filename)
+
+        section = 'TerraMatch corrections v2'
+        easting = config.getfloat(section, 'EastingShift')
+        northing = config.getfloat(section, 'NorthingShift')
+        elevation = config.getfloat(section, 'ElevationShift')
+        heading = config.getfloat(section, 'HeadingShift')
+        roll = config.getfloat(section, 'RollShift')
+        pitch = config.getfloat(section, 'PitchShift')
+
+        t0 = math.cos(heading * 0.5)
+        t1 = math.sin(heading * 0.5)
+        t2 = math.cos(roll * 0.5)
+        t3 = math.sin(roll * 0.5)
+        t4 = math.cos(pitch * 0.5)
+        t5 = math.sin(pitch * 0.5)
+        qw = t0 * t2 * t4 + t1 * t3 * t5
+        qx = t0 * t3 * t4 - t1 * t2 * t5
+        qy = t0 * t2 * t5 + t1 * t3 * t4
+        qz = t1 * t2 * t4 - t0 * t3 * t5
+
```

Comment:
This TMS file seems to follow the convention Z(heading)X(roll)Y(pitch), which translates into the following quaternion (according to https://en.wikipedia.org/wiki/Conversion_between_quaternions_and_Euler_angles#cite_note-nasa-rotation-2 ) :

```
h =  math.radians(heading * 0.5)
r =  math.radians(roll * 0.5)
p =  math.radians(pitch * 0.5)

ch = math.cos(h)
sh = math.sin(h)
cr = math.cos(r)
sr = math.sin(r)
cp = math.cos(p)
sp = math.sin(p)

qw = - sh * sr * sp + ch * cr * cp
qx = - sh * cr * sp + ch * sr * cp
qy = + sh * sr * cp + ch * cr * sp
qz = + ch * sr * sp + sh * cr * cp
```

but that needs testing for verification !